src/test/java/org/broadinstitute/hellbender/tools/picard/analysis/directed/CollectTargetedPcrMetricsIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.tools.picard.analysis.directed;
+        import org.broadinstitute.hellbender.CommandLineProgramTest;
+        import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+        import org.broadinstitute.hellbender.utils.test.BaseTest;
+        import org.testng.annotations.Test;
+
+        import java.io.File;
+        import java.io.IOException;
+
+/**
+ * Created by dkling on 10/14/15.
+ */
+
+public final class CollectTargetedPcrMetricsIntegrationTest extends CommandLineProgramTest {
+    private static final File TEST_DATA_PATH = new File(getTestDataDir(), "picard/analysis/directed/CollectTargetedPcrMetrics/");
+
+
+    @Test
+    public void testCollect () throws IOException {
+
+        final File input = new File(TEST_DATA_PATH, "first5Kamod_reordered.bam");
+        final File amplicon_intervals = new File(TEST_DATA_PATH, "intervals_b37_20_1.interval_list");
+        final File target_intervals = new File(TEST_DATA_PATH, "intervals_b37_20_1.interval_list");
+        final File expectedFile = new File(TEST_DATA_PATH, "PCRMetrics_new_bam.txt");
+        final File outfile = BaseTest.createTempFile("PCRMetrics", ".txt");
+        final File reference = new File(largeFileTestDir, "human_g1k_v37.20.21.fasta");
+        final File pertargetcoverage = new File(TEST_DATA_PATH, "pcr_metrics_pertarg_new.txt");
```

Comment:
Do I need to change this line?  
IntegrationTestSpec.assertEqualTextFiles(outfile,expectedFile, "#");  If yes, how shall I modify it?

Thanks,
-D